vendor/k8s.io/kubernetes/pkg/client/restclient/client.go

```diff
@@ -222,9 +222,11 @@ func (c *RESTClient) Verb(verb string) *Request {
 	backoff := c.createBackoffMgr()
 
 	if c.Client == nil {
-		return NewRequest(nil, verb, c.base, c.versionedAPIPath, c.contentConfig, c.serializers, backoff, c.Throttle)
+		return NewRequest(nil, verb, c.base, c.versionedAPIPath, c.contentConfig,
```

Comment:
> Timeout should be a helper on Request, vs adding something to the constructor. There are lots of reasons to set timeouts per request type.

@smarterclayton `Timeout` is already a separate method on `Request`. I was thinking rather that we'll copy the value passed through client, and if someone will actually use `Timeout` that will override that this. What I had in mind is that I wanted to have a global solution that will just pass the timeout. It's reasonable to use `Timeout` method here instead of modifying the constructor, though.